src/lando/main/models/commit_map.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from django.db import models
+
+from lando.main.models import BaseModel
+from lando.main.models.repo import Repo
+
+
+class CommitMap(BaseModel):
+    """Map a git hash to an hg hash, based on a specific repo."""
+
+    HGMO_PUSHLOG_TEMPLATE = "https://hg.mozilla.org/{}/json-pushes"
+    REPO_MAPPING = (("firefox", "mozilla-unified"),)
+
+    git_hash = models.CharField(default="", unique=True)
+    hg_hash = models.CharField(default="", unique=True)
+
+    # NOTE: This value is set because multiple Lando repos can map to a single hg repo.
+    # This is because currently a separate repo object is created for each branch of a
+    # git repo, though those might map to a single hg repo (e.g., mozilla-unified).
+    git_repo_name = models.CharField(default="")
+
+    @classmethod
+    def get_hg_repo_name(cls, git_repo_name: str) -> str:
+        """Return mapped repo name or `git_repo_name` by default."""
+        return dict(cls.REPO_MAPPING).get(git_repo_name, git_repo_name)
```

Comment:
Could we make the mapping a dict to start with?

Reply:
See https://github.com/mozilla-conduit/lando/pull/340#discussion_r2159131505.